armbian/base/scripts/bbb-config.sh

```diff
@@ -14,16 +14,17 @@ usage: bbb-config.sh [--version] [--help]
 assumes Redis database running to be used with 'redis-cli'
 
 possible commands:
-  enable    <bitcoin_ibd_clearnet|dashboard_hdmi|dashboard_web|wifi|autosetup_ssd|
+  enable    <bitcoin_incoming|dashboard_hdmi|dashboard_web|wifi|autosetup_ssd|
```

Comment:
Should `bitcoin_ibd_clearnet` be removed? That seems to be a breaking change.

Reply:
This command has been [moved](https://github.com/digitalbitbox/bitbox-base/blob/ac0307e654299102dee9a414f584f2751f4b424c/armbian/base/scripts/bbb-config.sh#L273) from `enable/disable` to the `set` part of the script previously, just updating the usage instructions.

Reply:
Your right. Did not look properly.